include/unistd.h

```diff
@@ -25,7 +25,7 @@ SOFTWARE.
 
 #ifndef _WIN32
 
-#pragma message("this unistd.h implementation is for Windows only!")
+#include <unistd.h>
```

Comment:
This file is from `win32ports` project, for best practice, I won't allow to change it in this repo.